### PR TITLE
[docs] Add some testing information

### PR DESCRIPTION
**Pull request by almet**

### CODE REVIEW

**Review by davidbgk — APPROVED** [review]
❤️

File: Makefile
Comment (yohanboniface) [inline Makefile:76]: Not sure why you removed this ?

Comment (davidbgk) [reply Makefile:76]: This is not cross-platform

Comment (almet) [reply Makefile:76]: It's because, with the ES modules you need a browser to run. Instead of using firefox directly, my proposal here is to print an URL, and then you can open it the way you want.

Would that work for you?

**Comment by almet** [comment]
But I'm sorry, I don't remember hitting the merge button here. Was waiting for @yohanboniface review first. Not sure when that happened 😳

File: Makefile
Comment (yohanboniface) [reply Makefile:76]: Hehe, not a big deal :)

I remember seeing that `open path/to/index.html` would be cross platform, no ? 

At least, I do use `make testjsfx`, so I'd prefer if it still works :)

Comment (davidbgk) [reply Makefile:76]: As far as I understand, it has been merged with `testjs` so it still works somehow but maybe the `mocha-phantomjs` call was still useful?

Comment (almet) [reply Makefile:76]: Ah, if `open` works that would be perfect, but I think it's not cross-platform. Does it work on your OS?

Comment (yohanboniface) [reply Makefile:76]: In mine, yes :)

Comment (almet) [reply Makefile:76]: > maybe the mocha-phantomjs call was still useful?

It wasn't working for me. If it's useful for somebody I can put it back of course :-)

Comment (yohanboniface) [reply Makefile:76]: No, that one is broken, it can be removed for sure

Comment (almet) [reply Makefile:76]: See #1472

**Pull request merged**
